src/main/java/uk/gov/pay/publicauth/filters/LoggingFilter.java

```diff
@@ -27,22 +24,23 @@ public void init(FilterConfig filterConfig) { }
     @Override
     public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) {
         Stopwatch stopwatch = Stopwatch.createStarted();
+        HttpServletRequest httpRequest = (HttpServletRequest)servletRequest;
```

Comment:
Space between `(HttpServletRequest)` and `servletRequest` please

Reply:
Done